portfolio/src/main/webapp/forum.html

```diff
@@ -55,4 +55,40 @@ <h2>Post comments, discussion, or thoughts here!</h2>
         </div>
     </body>
 
+    <!-- jQuery functions -->
+    <script type="text/javascript">
+
+      // checks if comment form contains content    
```

Comment:
Can you make this a bit more specific:

"Disable submit button if form is empty."

Reply:
updated this!